src/components/ArticleList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import { ArticleCard } from "./ArticleCard";
+
+export const ArticleList = () => {
+    const [articles, setArticles] = useState([]);
+
```

Comment:
You can make use of the `isLoading` pattern here to give users (especially those on a slower connection) an indication that something is actually happening behind the scenes